File: FlexServ Backend/SpringBoot/AdminService/src/main/java/com/flexserv/dto/response/AdminResponse.java
package com.flexserv.dto.response;

import java.time.LocalDateTime;

import com.flexserv.entity.Role;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class AdminResponse {

    private Long id;

    private String name;

    private String email;

    private String phone;

    private Role role;

    private String department;

    private Boolean isActive;

    private LocalDateTime createdAt;
}

File: FlexServ Backend/DotNet/AdminService/Dtos/AdminResponse.cs
using AdminService.Models;
using System;

namespace AdminService.Dtos
{
    public class AdminResponse
    {
        public long Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Email { get; set; } = string.Empty;
        public string Phone { get; set; } = string.Empty;
        public Role Role { get; set; }
        public string Department { get; set; } = string.Empty;
        public DateTime? CreatedAt { get; set; }

        public AdminResponse() { }

        public AdminResponse(long id, string name, string email, string phone, Role role, string department, DateTime? createdAt)
        {
            Id = id;
            Name = name;
            Email = email;
            Phone = phone;
            Role = role;
            Department = department;
            CreatedAt = createdAt;
        }
    }
}
